code/modules/core_implant/cruciform/machinery/wall_obelisk.dm

```diff
@@ -66,27 +66,25 @@
 		use_power = 2
 
 		var/to_fire = max_targets
-		for(var/A in affected)
-			if(istype(A, /obj/structure/burrow))
-				var/obj/structure/burrow/burrow = A
-				if(!burrow.obelisk_around)
-					burrow.obelisk_around = any2ref(src)
-			else if(istype(A, /mob/living/carbon/superior_animal))
-				var/mob/living/carbon/superior_animal/animal = A
+		for(var/mob/living/carbon/superior_animal/superior_mob in affected)
+			if(istype(superior_mob, /mob/living/carbon/superior_animal))
```

Comment:
This check is irrelevant, as the type of the var in the for loop automatically typechecks.

Also, you dont need to remove the burrow code. Thats part of how the obelisk works.